src/wmtk/operations/tet_mesh/TetSplitWithTags.cpp

```diff
@@ -8,8 +8,9 @@
 
 namespace wmtk::operations {
 
-void OperationSettings<tet_mesh::TetSplitWithTags>::initialize_invariants(const TetMesh& m)
+void OperationSettings<tet_mesh::TetSplitWithTags>::initialize_invariants()
 {
+    const TetMesh& m = static_cast<TetMesh&>(invariants.mesh());
```

Comment:
here's how we should read the mesh from the held invariantcollection from now on